src/VoyagerServiceProvider.php

```diff
@@ -270,22 +269,20 @@ public function registerGates()
         // otherwise it will throw an error because no database
         // connection has been made yet.
         try {
-            if (Schema::hasTable('data_types')) {
```

Comment:
What's the reasoning behind removing this check?

Reply:
Just for try i remove that if, now are ok